base/dcp_client.go

```diff
@@ -189,9 +196,17 @@ func (dc *DCPClient) close() {
 }
 
 func (dc *DCPClient) initAgent(spec BucketSpec) error {
-	connStr, err := spec.GetGoCBConnString(&GoCBConnStringParams{
-		KVPoolSize: GoCBPoolSizeDCP,
-	})
+	defaultValues := &GoCBConnStringParams{}
+	if dc.Serverless {
```

Comment:
Similar to the previous case, I don't think we need a 'Serverless' property on the DCPClient.  We just need to set these values appropriately in the spec and retrieve them from there.